radio/src/pulses/pulses.h

```diff
@@ -84,6 +84,16 @@ enum ModuleFlag
   #define IS_DSM2_SERIAL_PROTOCOL(protocol)  (0)
 #endif
 
+#if defined(MULTIMODULE)
+  #define IS_MULTIMODULE_PROTOCOL(protocol)  (protocol==PROTO_MULTIMODULE)
+  #if !defined(DSM2)
+     #error You need to enable DSM2 = PPM for MULTIMODULE support
```

Comment:
Would it not be better to force it in the Makefile?
Move the 
``` ifeq ($(MULTIMODULE), YES)```
block before the DSM2 one, and add `DSM2=PPM` in it?